node/src/accountant/mod.rs

```diff
@@ -3381,429 +4099,450 @@ mod tests {
+        let expected_rowid = 45623;
+        let pending_payable_dao = PendingPayableDaoMock::default()
+            .fingerprints_rowids_params(&fingerprints_rowids_params_arc)
+            .fingerprints_rowids_result(TransactionHashes {
+                rowid_results: vec![(expected_rowid, expected_hash)],
+                no_rowid_results: vec![],
+            });
+        let payable_dao = PayableDaoMock::new()
+            .mark_pending_payables_rowids_params(&mark_pending_payables_rowids_params_arc)
+            .mark_pending_payables_rowids_result(Ok(()));
+        let system =
+            System::new("accountant_processes_sent_payables_and_schedules_pending_payable_scanner");
+        let mut subject = AccountantBuilder::default()
+            .bootstrapper_config(bc_from_earning_wallet(make_wallet("some_wallet_address")))
+            .payable_daos(vec![ForPayableScanner(payable_dao)])
+            .pending_payable_daos(vec![ForPayableScanner(pending_payable_dao)])
+            .build();
+        let pending_payable_interval = Duration::from_millis(55);
+        subject.scan_schedulers.pending_payable.interval = pending_payable_interval;
+        subject.scan_schedulers.pending_payable.handle = Box::new(
+            NotifyLaterHandleMock::default()
+                .notify_later_params(&pending_payable_notify_later_params_arc),
+        );
+        let expected_payable = PendingPayable::new(expected_wallet.clone(), expected_hash.clone());
+        let sent_payable = SentPayables {
+            payment_procedure_result: Ok(vec![ProcessedPayableFallible::Correct(
+                expected_payable.clone(),
+            )]),
+            response_skeleton_opt: None,
+        };
+        let addr = subject.start();
+
+        addr.try_send(sent_payable).expect("unexpected actix error");
+
+        System::current().stop();
+        system.run();
+        let fingerprints_rowids_params = fingerprints_rowids_params_arc.lock().unwrap();
+        assert_eq!(*fingerprints_rowids_params, vec![vec![expected_hash]]);
+        let mark_pending_payables_rowids_params =
+            mark_pending_payables_rowids_params_arc.lock().unwrap();
+        assert_eq!(
+            *mark_pending_payables_rowids_params,
+            vec![vec![(expected_wallet, expected_rowid)]]
+        );
+        let pending_payable_notify_later_params =
+            pending_payable_notify_later_params_arc.lock().unwrap();
+        assert_eq!(
+            *pending_payable_notify_later_params,
+            vec![(ScanForPendingPayables::default(), pending_payable_interval)]
+        );
+        // The accountant is unbound here. We don't use the bind message. It means we can prove
+        // none of those other scan requests could have been sent (especially ScanForNewPayables,
+        // ScanForRetryPayables)
+    }
+
+    #[test]
+    fn zero_sent_payables_so_the_payable_scan_is_rescheduled_without_scanning_for_pending_payables()
```

Comment:
I'd be happy if we have something other than zero, as it can be confused as empty vector.

```rust
no_payables_left_the_node_so_payable_scan_is_rescheduled_as_pending_payable_scan_was_omitted
```